Example/src/SharedElementTransition/CustomTransition.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { View, Button } from 'react-native';
+import { ParamListBase } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import Animated, {
+  SharedTransition,
+  withSpring,
+} from 'react-native-reanimated';
+import { StackScreenProps } from '@react-navigation/stack';
+
+const Stack = createNativeStackNavigator();
+
+const transition = SharedTransition.custom((values: any) => {
```

Comment:
It works without `any` here.
```suggestion
const transition = SharedTransition.custom((values) => {
```